opentelemetry-sdk/Cargo.toml

```diff
@@ -37,7 +37,7 @@ rustdoc-args = ["--cfg", "docsrs"]
 
 [dev-dependencies]
 bincode = "1.2"
-criterion = "0.4.0"
+criterion = "0.5.0"
```

Comment:
Not sure this really belongs in this PR. Also, might run into MSRV issues.

Reply:
Ah, sorry... This is a "workflow issue" on my end. (pushed to the wrong branch.)